### ISSUE TITLE
Add NFC Boltcard compatibility

### ISSUE THREAD

**Issue by ElucGeek** [nbd-wtf/obw#9]
Hello,
is it on the roadmap to add NFC Boltcard compatibility? Would be awesome.
Best regards

**Comment by fiatjaf** [comment]
This is a great idea. Do you know where are the specs for that?

**Comment by Bashy** [comment]
> add NFC Boltcard compatibility
I'm not sure to understand,
Make OBW able to withdraw funds on a NFC Boltcard?

**Comment by ElucGeek** [comment]
> > add NFC Boltcard compatibility
> 
> @ElucGeek I'm not sure to understand, make OBW able to withdraw funds from a NFC Boltcard?

Exactly. Breez can do that, you create an invoice from the PoS, then you tap the card on the back of the phone where is the NFC antenna, and it's paid!

More information here, if you are LUD17 LNURLW compatible it should be easy to integrate according to this website:
https://www.boltcard.org/

There is a Telegram group as well, sorry I'm not dev I cannot help much more.

**Comment by fiatjaf** [comment]
Have you tried to do that on OBW? It has NFC support already.
I guess it just needs LUD-17 then?

**Comment by ElucGeek** [comment]
I just tried on Android, it's not working, it asks which app I want to open the card with and I can choose Breez, WoS or VoltPay which are compatible.
When I scan from one of these app directly it works without asking to choose an app.
